Formula/cubelib.rb

```diff
@@ -0,0 +1,28 @@
+class Cubelib < Formula
+  desc "Cube, is a performance report explorer for Scalasca and Score-P"
+  homepage "https://scalasca.org/software/cube-4.x/download.html"
+  url "https://apps.fz-juelich.de/scalasca/releases/cube/4.4/dist/cubelib-4.4.2.tar.gz"
+  sha256 "843335c7d238493f1b4cb8e07555ccfe99a3fa521bf162e9d8eaa6733aa1f949"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
+    # Some checks uses output from other checks
+    ENV.deparallelize
+    system "make", "check"
```

Comment:
Are these checks required for the install?

Reply:
No

Reply:
done